enry_stage.py

```diff
@@ -0,0 +1,15 @@
+from enry import get_languages
+import os
+
+def run_enry_by_file(CWD, base_dir_path):
```

Comment:
let's add docstrings everywhere
`CWD` is not so great naming - let's use meaningful names

Reply:
let's rename `base_dir_path` to `target_dir` or something like this - it will show that we will launch `enry` on target directory